src/software/firmware/srcs/respirator.cpp

```diff
@@ -126,13 +126,13 @@ void setup() {
     waitForInMs(4000);
 
     // escBlower start
-    hardwareTimer3->setCaptureCompare(TIM_CHANNEL_ESC_BLOWER, Angle2MicroSeconds(170),
+    hardwareTimer3->setCaptureCompare(TIM_CHANNEL_ESC_BLOWER, BlowerAngle2MicroSeconds(170),
                                       MICROSEC_COMPARE_FORMAT);
     DBG_DO(Serial.println("Blower is running.");)
 
     // Init the watchdog timer. It must be reloaded frequently otherwise MCU resests
-    IWatchdog.begin(WATCHDOG_TIMEOUT);
-    IWatchdog.reload();
+    // IWatchdog.begin(WATCHDOG_TIMEOUT);
```

Comment:
Is it kept commented on purpose?

Reply:
No! We did a live code review and @jabby is already correcting this one, and few other minor issues